index.html

```diff
@@ -1,12 +1,147 @@
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
+     <link href="https://fonts.googleapis.com/css2?family=Poppins" rel="stylesheet">
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>main</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <header>
+      <section class="container" >
+        <div class="navheader">
+          <div class="black">
+            <ul>
+              <li><a href="#"><img src="images/twitterlogo.png" alt="twitterlogo"></a></li>
+              <li><a href="#">English</a></li>
+              <li><a href="#">Mypage</a></li>
+              <li><a href="#">Log out</a></li>
+            </ul>
+          </div>
+        </div>
+    <nav>
+    <div class ="navbar1">
+        <img src="images/menu.png" id="menu-icon" alt="humbergur images">
+        <div class="desk-menu" id="deskMenu">
+            <div class="desk-menu-inside">
+             <div class="logo"><img src="images/africalogo.webp" alt="logo images"></div>
+             <ul>
+              <li ><a class="menu" href="./index.html">Home</a></li>
+               <li ><a class="menu" href="./about.html">About</a></li>
+               <li ><a class="menu" href="#about.html">Program</a></li>
+               <li ><a class="menu" href="#contact-me">Join</a></li>
+               <li ><a class="menu" href="#about-me">Sponser</a></li>
+               <li ><a class="menu" href="#contact-me">Register</a></li>
+               <li ><a href="index.html"> <button class="menu-btn"> Presentation</button></a></li>
+            </ul>
+           </div>
+        </div>
     
+    <div class="mobile-menu" id="mobileMenu">
+        <img id="menu-close" src="images/closeIcon.png" alt="close icon">
+        <ul class="mob-menu">
+          <li class="menu"><a class="moblink-menu" href="./index.html">Home</a></li>
+          <li class="menu"><a class="moblink-menu" href="./about.html">About</a></li>
+          <li class="menu"><a class="moblink-menu" href="#about.html">Program</a></li>
+          <li class="menu"><a class="moblink-menu" href="#contact-me">Join</a></li>
+          <li class="menu"><a class="moblink-menu" href="#about-me">Sponser</a></li>
+          <li class="menu"><a class="moblink-menu" href="#contact-me">Register</a></li>
+        </ul>
+    </div>
+    
+     </div>
+  </nav>
+    <div class="front">
+        <div class="front1">
+        <h2>"hello! developers"</h2>
+        <h1>SOFTWARE DEVELOPERS <br> GLOBAL SUMMIT 2023</h1>
```

Comment:
**OPTIONAL** - In order to give your site a better HTML structure, it is very important that your `hx` tags follow a proper hierarchy i.e an `h2` tag follows after an `h1` tag, an `h3` tag comes after using an `h2` tag and not vice versa. Please fix this issue 👍 👍